### DISCUSSION TITLE
Timelapse for Construction

### DISCUSSION THREAD

**Discussion by irishpeck** [gopro/labs#1129] [General]
Hi Guys,

I have a GoPro Hero 10 Black and want to try and modify it to record some timelapses of our construction site builds. 

My question is this I have installed the Firmware update on the GoPro and can see that this is working however when I scan a QR code I don't actually know what I am meant to do. Can someone give me a quick overview what I should do in order to get this timelapse to work. Thanks in advance

**Comment by Espusen** [comment]
Hi there

You can have a look at my [Knowledge library](https://www.altofocus.co.uk/gopro-labs). Its £20 and you get explainer videos and over a dozen different Timelapse codes for labs.
You can also easily (and very cheaply) upload the images to an editor and have it edited professionally! 
If it is no use to you, let me know and I will refund you.
https://www.altofocus.co.uk/

**Reply by irishpeck** [reply]
Hi There Espusen, I will come back to this it might just be more useful to use your facility to get this going. I did manage to get a timelapse of sorts going but then it did run out of battery overnight so that obviously won't work. Another question I have you might be able to help is regarding Live video functionality. Can I setup one camera to do Timelapse and another with live video functionality and actually login to it at different points of the night to see what is happening?

**Reply by Espusen** [reply]
Hi there

Live video is possible for sure but the biggest issue you will have with remote/unattended cameras is and always will be power and potential overheating.
It can possibly be scheduled to run live for, say 1 hour per day but I have not done any testing.

What are you trying to achieve? CCTV type monitoring? There are many other solutions available for this I think. If you want access to uploaded images from the gopro that is definitely achievable but only with an external programmer attached to the GoPro. Just message me if you want to know more.

**Comment by dnewman-gpsw** [comment]
There are a lot of options. You can experiment starting here https://gopro.github.io/labs/control/construction/  

However, the HERO10 is not the ideal choice for the use scripts that wake a sleep the camera over long periods of time, this is better with H12 or H13.  However all black edition cameras can be externally powered, and record timelapse definitely.  For a hybrid approach, that only timelapses during the day, you can try https://gopro.github.io/labs/control/dailytl/

**Reply by irishpeck** [reply]
Thank you for the info sir. For now just testing on the H10 but will be looking to upgrade the cameras to the newest offering when launching the timelapses.
